app/models/calendar.rb

```diff
@@ -272,4 +273,12 @@ def check_source_reachable
   rescue CalendarImporter::Exceptions::UnsupportedFeed => e
     errors.add :source, 'Unable to autodetect calendar format, please pick an option from the list below'
   end
+
+  def clear_status_on_source_change
+    return unless source_changed?
+
+    self.critical_error = nil
+    self.notices = nil
+    self.last_import_at = nil
+  end
```

Comment:
This looks like it works though is it missleading to imply the issue has been fixed when it's not been tested yet? @kimadactyl  is this what you wanted?

I suppose the alternative is not clearing the issue and making it clear that you need to wait for the importer to run to findout if the changes have been effective?

happy for this to go in, just double checking before we commit to it.

Reply:
yeah hmmm lots of ways you could argue this

i think it prob makes most sense to change the messages to something like "Source URL changed, waiting for queue" and not to blank the last_import?

Reply:
If you feel this is the desired behaviour then the A/C need to be (possibly discussed) and updated